dev/com.ibm.ws.concurrency.policy/.classpath

```diff
@@ -4,5 +4,6 @@
 	<classpathentry kind="con" path="aQute.bnd.classpath.container"/>
 	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-1.8"/>
 	<classpathentry kind="src" path="/com.ibm.ws.threading"/>
+	<classpathentry kind="src" path="/com.ibm.ws.logging.core"/>
```

Comment:
Both this classpath entry and the one above should be added in bnd.bnd, not the eclipse classpath file.